.golangci.yaml

```diff
@@ -0,0 +1,23 @@
+linters:
+  disable-all: true
+  enable:
+    # errcheck and govet are part of default setup and should be included but give too many errors now
+    # once errors are fixed, they should be enabled here:
+    #- errcheck
+    - gosimple
+    #- govet
+    - ineffassign
+    - staticcheck
+    - unused
+    - gofmt
+linters-settings:
+  gofmt:
+    rewrite-rules:
+      - pattern: 'a[b:len(a)]'
+        replacement: 'a[b:]'
+issues:
+  exclude-dirs-use-default: false  # recommended by docs https://golangci-lint.run/usage/false-positives/
+  exclude:
+    # Excluded because they are currently failing. Once fixed, these exceptions should be removed from here:
+    - "SA1019: tfexec.State is deprecated: The -state CLI flag is a legacy flag and should not be used. If you need a different state file for every run, you can instead use the local backend. See https://github.com/hashicorp/terraform/issues/25920#issuecomment-676560799"
+    - "SA1012: do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use"
```

Comment:
Both of these have `//lint:ignore SA1012` comments above them. Is there a way to selectively ignore rules?